README.rst

```diff
@@ -21,23 +23,98 @@ pythx
         :target: https://coveralls.io/github/dmuhs/pythx?branch=master
 
 
+PythX is a library for the Mythx_ smart contract security analysis platform.
+
+.. code-block::
+
+        Usage: pythx [OPTIONS] COMMAND [ARGS]...
+
+        Options:
+        --help  Show this message and exit.
+
+        Commands:
+        check    Submit a new analysis job based on source code, byte code, or...
+        login    Login to your MythX account
+        logout   Log out of your MythX account
+        openapi  Get the OpenAPI spec in HTML or YAML format
+        ps       Get a greppable overview of submitted analyses
+        refresh  Refresh your MythX API token
+        report   Check the detected issues of a finished analysis job
+        status   Get the status of an analysis by its UUID
+        top      Display the most recent analysis jobs and their status
+        version  Print version information of PythX and the API
+
+
+By default, PythX comes enabled with a trial user:
+
+.. code-block::
+
+        $ pythx login
+        Please enter your Ethereum address [0x0000000000000000000000000000000000000000]:
+        Please enter your MythX password [trial]:
+        Successfully logged in as 0x0000000000000000000000000000000000000000
+
+
+Submit an Solidity source file for analysis:
+
+.. code-block::
+        $ pythx check -sf test.sol
```

Comment:
Does the `-sf` flag stand for source file?
Would the command to submit a bytecode for analysis be `pythx check -bc 0x...`?

This can be done later on, but it would be nice expanding the help manual to include clarify check options.

Reply:
Currently `sf` for Solidity source files and `-bf` for creation bytecode files are supported. I wanted to avoid passing the bytecode directly through the CLI because it can get pretty large